samples/persistent_subscriptions.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+from uuid import uuid4
+
+from esdbclient import EventStoreDBClient, NewEvent, StreamState
+
+stream_name = str(uuid4())
+
+client = EventStoreDBClient(uri="esdb://localhost:2113?tls=false")
+
+# region create-persistent-subscription-to-stream
+client.create_subscription_to_stream(
+    stream_name=stream_name,
+    group_name="subscription-group",
+)
+# endregion create-persistent-subscription-to-stream
+
+event_data = NewEvent(
+    type="some-event",
+    data=b'{"id": "1", "important_data": "some value"}',
+)
+
+client.append_to_stream(
+    stream_name=stream_name,
+    current_version=StreamState.ANY,
+    events=event_data,
+)
+
+# region subscribe-to-persistent-subscription-to-stream
+subscription = client.read_subscription_to_stream(
+    stream_name=stream_name,
+    group_name="subscription-group",
+)
+
+try:
+    for event in subscription:
+        try:
+            print(
+                f"handling event {event.type} with"
+                f" retryCount {event.retry_count}"
+            )
+            subscription.ack(event_id=event.id)
+            break
+        except Exception as ex:
+            print(f"handling failed with exception {ex}")
+            subscription.nack(event_id=event.id, action="park")
+except BaseException as e:
+    print(f"Subscription was dropped. {e}")
+# endregion subscribe-to-persistent-subscription-to-stream
+
+
+event_data = NewEvent(
+    type="some-event",
+    data=b'{"id": "1", "important_data": "some value"}',
+)
+
+client.append_to_stream(
+    stream_name=stream_name,
+    current_version=StreamState.ANY,
+    events=event_data,
+)
+
+GROUP_NAME = str(uuid4())
+
+client.create_subscription_to_all(
+    group_name=GROUP_NAME,
+)
+
+# region subscribe-to-persistent-subscription-to-all
+subscription = client.read_subscription_to_all(
+    group_name=GROUP_NAME,
+)
+
+try:
+    for event in subscription:
+        print(
+            "handling event {event.type} with"
+            " retryCount {event.retry_count}"
+        )
+        subscription.ack(event_id=event.id)
+        break
+except BaseException as e:
```

Comment:
Again, probably, either you want to reraise the `BaseException` or you want to use `Exception` here? But I'm not sure what you are trying to express.